Packs/Illumio/Integrations/IllumioCore/IllumioCore.yml

```diff
@@ -261,6 +261,9 @@ script:
     - contextPath: Illumio.VirtualService.service_addresses
       description: Service addresses of the virtual service.
       type: Unknown
+    - contextPath: Illumio.VirtualService.already_exists
```

Comment:
I don't think we need these in the output, it's confusing. Just add a log line/note or equivalent.

Reply:
This key is required in the case of a playbook because In the playbook we're doing certain condition checks where any object if already exists then we need to skip certain tasks. There is no straight way that we can get the Illumio script access in the playbook that's why we're adding this key in the context path.

Reply:
Can you provide an example of this? What's the case where a task needs to be skipped if the object exists?

Reply:
if a virtual service already exists then we'll skip the ruleset & rule creation task and directly create the enforcement boundary.

Reply:
I don't think that's correct - the virtual service existing doesn't imply that the ruleset also exists. The cost of the ruleset/rule tasks is relatively low if the objects already exists (two API calls and the turnaround time for a task run) but if the ruleset is skipped and doesn't exist then we potentially block traffic that the user is expecting to be allowed.